content/applications/services/helpdesk/advanced/after_sales.rst

```diff
@@ -1,67 +1,218 @@
+button), or sent in an email by clicking :guilabel:`Send`.
 
-.. image:: after_sales/aftersales6.png
+.. note::
+   All followers of the ticket will be added as recipients to the email. Additional recipients can
+   be added to the email as well.
+
+.. image:: after_sales/after-sales-coupon-email.png
    :align: center
-   :height: 300
+   :alt: View of an email draft window with coupon code.
+
+Once a coupon code has been generated, a smart button will be added to the top of the Helpdesk
+ticket. Click here to view the coupon code, expiration date, and additional information.
+
+.. image:: after_sales/after-sales-coupon-smart-button.png
+   :align: center
+   :alt: View of the smart buttons on a ticket focusing on the coupon button.
 
 .. seealso::
    - `Coupons <https://www.odoo.com/slides/slide/coupon-programs-640?fullscreen=1>`_
+
+Facilitate a product return with a reverse transfer
+===================================================
+
+Returns are completed through *Reverse Transfers*, which generate new warehouse operations for the
+returning products. Click the :guilabel:`Return` button at the top of a ticket to open the
+:guilabel:`Reverse Transfer` pop-up window.
+
+.. image:: after_sales/after-sales-return-button.png
+   :align: center
+   :alt: View of a Helpdesk ticket with the Return button highlighted.
+
+.. note::
+   The :guilabel:`Return` button will only appear on a ticket if the customer selected has recorded
+   deliveries in the database.
+
+By default, the quantity will match the validated quantity from the delivery order. Update the
+quantity field if necessary.
+
+.. image:: after_sales/after-sales-reverse-transfer.png
+   :align: center
+   :alt: View of a Reverse Transfer creation page.
+
+Click :guilabel:`Return` to confirm the return. This generates a new warehouse operation for the
+incoming returned product(s). A :guilabel:`Return` smart button will then be added to the top of
+the Helpdesk ticket.
+
+.. image:: after_sales/after-sales-return-smart-button.png
+   :align: center
+   :alt: View of the smart buttons on a ticket focusing on returns.
+
+.. seealso::
+   - :doc:`/applications/sales/sales/products_prices/returns`
+
+Send products for repair from a ticket
+======================================
+
+If the :guilabel:`Helpdesk` ticket is reporting an issue caused by a faulty or broken product, a
+repair order can be created directly from the ticket. To create a new repair order, click
+:guilabel:`Repair`.
+
+.. image:: after_sales/after-sales-repair-reference.png
+   :align: center
+   :alt: View of a Repair Reference page.
+
+If a product was specified in the :guilabel:`Product` field on the ticket, it will be automatically
+added to the :guilabel:`Product to Repair` field on the newly created *Repair Reference*. If
+applicable, add the :guilabel:`Sale Order` and :guilabel:`Return` from which the faulty product
+originates.
```

Comment:
```suggestion
If a product was specified in the :guilabel:`Product` field on the ticket, it will be automatically
added to the :guilabel:`Product to Repair` field on the newly created :guilabel:`Repair Reference`. If
applicable, add the :guilabel:`Sale Order` and :guilabel:`Return Order` from which the faulty product
originates.
```